_static/css/custom.css

```
@@ -1844,3 +1844,11 @@ p + .classref-constant {
 #godot-giscus {
     margin-bottom: 1em;
 }
+
+/** Center videos embedded using the sphinxcontrib-video plugin.
+ * That plugin makes assumptions about `align-center` and `align-left` that our
+ * theme does not follow, so we set the `align-default` class instead to avoid
+ * collisions and have full control of how videos are aligned. */
+.align-default {
+    text-align: center
```

```suggestion
    text-align: left
```
If we would rather keep exact parity with existing behavior, we can do this. I chose to center the videos because:
1. The original report of this problem on RC implied to me that centering might be the desired behavior (even though in 4.3 our videos *were* aligned left!)
2. Images using the `.. figure:` role are generally aligned center, images using the `.. image:` role are generally aligned left (implicitly). So there is *some* precedent for center alignment.
3. I think it looks better :)

I don't really care all that much one way or another. This is something we should perhaps standardize at some point, along with guidelines about whether/when to use `.. figure:` or `.. image:`.

---

This broke tables in the class reference (and possibly elsewhere) I think, they're now all centered and it really isn't readable